bin/main.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./../lib/Game_controller')
+require_relative('./../lib/game')
```

Comment:
these paths aren't working

Reply:
First bug: What do you mean with arent working?
Second bug: I fixed i think in my last commit.
About Feature branch: im just fixing bugs thats not a new feature, that's why i haven't create a feature branch i think.